pontoon/settings/base.py

```diff
@@ -191,6 +191,7 @@ def _site_url():
 TEMPLATES = [
     {
         'BACKEND': 'django_jinja.backend.Jinja2',
+        'NAME': 'jinja2',
```

Comment:
Saw that in django-jinja's documentation, its own test project also. Used for django.template.engines (engines['jinja2']) which returns `env` (environment). Don't know if it will be useful in pontoon in future, I solved problem without using it. Should I remove it?